doc/data-structures/in-database.md

```diff
@@ -160,9 +160,13 @@ metadata: information about the layer
   dataCalculationDescription: String
 ```
 
-# Attributes
+## Attributes
+
+Attributes are values with specified id/name, date when the values were obtained or created and geographical area where they are from.  
+This values are stored in two collections based on type of geographical information and where it is stored.
+For value for certain point on map with geographical information included, we have `pointAttributes` collection structured exactly like features in geojson file.
 
-Attributes are stored in attributes collection. Each document is created for one geographical unit in official levels and for given date. They are used in [map view](../UI/map.md), linked by 'attribute' in mapLayers collection and 'attributeId' in attributes collection.  
+In case of static set of geographical features, e.g. provinces and districts in country or a set of areas of interest, we store the values in `featureAttributes` collection in database. They are used in [map view](../UI/map.md), linked by 'attribute' in mapLayers collection and 'attributeId' in attributes collection.  
```

Comment:
...in attributes collection. 

Should be there featureAttributes collection?

Reply:
as there are no "``" I took it for description of the collection, not specific name & this will be also massively changed before considering merge to main